pom.xml

```diff
@@ -103,6 +103,12 @@
       <artifactId>javax.json</artifactId>
       <version>1.0.4</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-compress</artifactId>
```

Comment:
This dependency doesn't seem to be used.

Reply:
It's used in another PR (extractor functionality) but snuck in here some how. I'll remove.